tutorials/tools/Text_Normalization_Tutorial.ipynb

```diff
@@ -0,0 +1,472 @@
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "7G-n2jYM0HpO"
+      },
+      "source": [
+        "eval_input_data = load_files([eval_text_file_path])\r\n",
+        "print(eval_input_data)"
+      ],
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "nYm8phOe1qa6"
+      },
+      "source": [
+        "print(\"Sentence level evaluation...\")\r\n",
+        "sentences_un_normalized, sentences_normalized = training_data_to_sentences(eval_input_data)\r\n",
+        "print(\"- Data: \" + str(len(sentences_un_normalized)) + \" sentences\")\r\n",
+        "sentences_prediction = normalize.normalize_nemo(sentences_un_normalized)\r\n",
+        "print(\"- Normalized. Evaluating...\")\r\n",
+        "sentences_accuracy = evaluate(sentences_prediction, sentences_normalized, sentences_un_normalized)\r\n",
+        "print(\"- Accuracy: \" + str(sentences_accuracy))"
+      ],
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "EwNGHovV2FYT"
+      },
+      "source": [
+        "print(\"Token level evaluation...\")\r\n",
+        "tokens_per_type = training_data_to_tokens(eval_input_data)\r\n",
+        "token_accuracy = {}\r\n",
+        "for token_type in tokens_per_type:\r\n",
+        "    print(\"- Token type: \" + token_type)\r\n",
+        "    tokens_un_normalized, tokens_normalized = tokens_per_type[token_type]\r\n",
+        "    print(\"  - Data: \" + str(len(tokens_un_normalized)) + \" tokens\")\r\n",
+        "    tokens_prediction = normalize.normalize_nemo(tokens_un_normalized)\r\n",
+        "    print(\"  - Normalized. Evaluating...\")\r\n",
+        "    token_accuracy[token_type] = evaluate(tokens_prediction, tokens_normalized, tokens_un_normalized)\r\n",
+        "    print(\"  - Accuracy: \" + str(token_accuracy[token_type]))\r\n",
+        "token_count_per_type = {token_type: len(tokens_per_type[token_type][0]) for token_type in tokens_per_type}\r\n",
+        "token_weighted_accuracy = [\r\n",
+        "    token_count_per_type[token_type] * accuracy for token_type, accuracy in token_accuracy.items()\r\n",
+        "]\r\n",
+        "print(\"- Accuracy: \" + str(sum(token_weighted_accuracy) / sum(token_count_per_type.values())))"
+      ],
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "lcvT3P2lQ_GS"
+      },
+      "source": [
+        "# Notes\n",
+        "\n",
+        "The current system expects well-formed sentences and word boundaries. The default expects a semiotic token to be surrounded by a non-word token. E.g. `A & E` will be detected as `VERBATIM`, however `A&E` will not be detected due to missing spaces around `&`. As an exercise, adjust the word boundary definition in `tools/text_normalization/tagger.py` to accommodate this too."
```

Comment:
link is missing for `tools/text_normalization/tagger.py`

Reply:
done